pkg/proxy/request.go

```diff
@@ -0,0 +1,138 @@
+package proxy
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/thedevsaddam/govalidator"
+	"go.uber.org/zap"
+
+	apisLoadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
+)
+
+const (
+	backendType     = "type"
+	distributedPods = "distributedPods"
+	testFile        = "testFile"
+	testData        = "testData"
+	envVars         = "envVars"
+	loadTestID      = "id"
+)
+
+//RequestValidator validates request body
+func httpValidator(r *http.Request) url.Values {
```

Comment:
The code below is the same as in https://github.com/hellofresh/kangal/blob/master/pkg/proxy/jmeter.go#L58
with few notes:
 - HTTP validator is not checking for different LoadTest ba
ckend types anymore.
 - HTTP validator is not making required distributedPods and testFile - that checks will be happened on domain side, on LoadTestSpec level:
https://github.com/hellofresh/kangal/pull/27/files#diff-a2c7b6b137a14498cd0e5e44b3740a5fR18-R28

That validation part will be also extracted on backend level check - because for example those fields are needed only on one load test backend(JMeter)